scalableminds/webknossos
# Add option to hide unmapped segments in segmentation tab

This PR adds the option to hide unmapped segments to the segmentation tab. The option is only available if a mapping is active.

### URL of deployed dev instance (used for testing):
- https://hideunmappedsegmentsinmergermode.webknossos.xyz

### Steps to test:
- Open a skeleton tracing and select a mapping
- When the mapping is loaded, the option to hide unmapped segments should appear and work as expected.
- Now open another skeleton tracing and start the merger mode. Using the new option you should see only mapped segments by skeletons. The other segments should be hidden.

### Issues:
- partly fixes #4398

------
